src/API/Entity.php

```diff
@@ -1407,14 +1402,7 @@ public function loadModelValues($model, $formData)
      */
     function simpleSave($model)
```

Comment:
I'll try to merge this behavior into the original BaseModel::save(). Instead of returning true on success, it'll return the ID.
Then, I'll replace all simpleSave() usages and just leave it as an alias for BC, so other projects won't break...
Sounds good?